docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -175,19 +176,22 @@ The sub-aim for this project is to create a 'power calculation' of how many pati
 5.2.1 Population description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  - cohort type
-  - cohort length
-  - age and sex structure
-  - time step
-  - fertility
-  - stratifications 
+  - Location: United States
```

Comment:
Should the cohort type be 18 years and older, since the clinical trial only focused on adults? Or do you want to include 15 years and older because the simulation will run for 3-5 years to initialize so that age group will be 18 years and older by the start of the sim?

Reply:
I set it to 15+ as our YLDs and YLLs due to MM start from 15 years old. Notice that simulants under 18 are not eligible for Isa.

Reply:
That makes sense.